Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Grid-Editor/index.md

```diff
@@ -0,0 +1,670 @@
+
+- **Grid Columns for Areas** - Overwrite the amount of columns used for the grid of Areas.
+- **Allow in area** - Determine wether this Block can be created inside Areas of other Blocks.
+
+### Area configuration
+
+> Missing screenshot of Area configuration
+
+**MISSING DESCRIPTION OF AREA CONFIG**
+Mention that the alias can be used to target this Area via the Layout Stylesheet.
+
+**Create Button Label** is to overwrite the Create Button Label of that Area.
+
+**Number of blocks** is for the total amount of Blocks in the Area.
+
+**TODO...**
+Mention that you can pick groups for allowance, in that case the required number is for Blocks of the group.
+Mention that you can pick individual Blocks, this Block can be picked and will be valid though its not allow for Areas.
+Individual Block validation(required number) is just for that block-type.
+
+
+### Advanced
+
+These properties are relevant when you working with custom views or complex projects.
+
+- **Custom view** - Overwrite the AngularJS view for the block presentation in the Content editor. Use this to make a more visual presentation of the block or make your own editing experience by adding your own AngularJS controller to the view.
+
+- **Custom stylesheet** - Pick your own stylesheet to be used for this block in the Content editor. By adding a stylesheet the styling of this block will become scoped. This means that the default backoffice styles are no longer present for the view of this block.
+
+- **Overlay editor size** - Set the size for the Content editor overlay for editing this block.
+
+- **Hide content editor** - Use this feature when you want to hide the ability to edit the content in a Block Editor. If you made a custom view that enables you to edit the content part of a Block, you might want to use this feature.
+
+## Editing Blocks
+
+When viewing a **Block Grid** editor in the Content section for the first time, you will be presented with the option to Add content.
+
+![Block List - Add Content](images/BlockListEditor_AddContent.png)
+> TODO: screenshot
+
+Clicking the Add content button brings up the **Block Catalogue**.
+
+![Block List - Setup](images/BlockListEditor_BlockPicker_simplesetup.jpg)
+> TODO: screenshot
+
+The Block Catalogue looks different depending on the amount of available Blocks and their catalogue appearance.
+
+![Block List - example setup from Umbraco.com](images/BlockListEditor_BlockPicker.jpg)
+> TODO: screenshot
+
+Click the Block Type you wish to create and a new Block will appear in the layout.
+
+More Blocks can be added to the layout by clicking the Add content button. Alternatively, use the Add content button that appears on hover to add new blocks between, besides or above existing Blocks.
+
+![Block List - Add Content](images/BlockListEditor_AddContentInline.jpg)
+> TODO: screenshot
+
+Blocks can be rearranged using the click and drag feature. Move them up or down to place them in the desired order.
```

Comment:
Add a note or tip to help mention that the grid can be configured to not allow blocks to be dropped in certain locations and will visually turn red @nielslyngsoe `

Reply:
You also need to mention the use of force left and force right and resizing blocks in an area with the resize handle.

Reply:
Okay, i have noted it.